AUTHORS.md

```diff
@@ -81,3 +81,4 @@ The following people have made contributions to this project:
 - [praerien (praerien)](https://github.com/praerien)
 - [Xin Zhang (zxdawn)](https://github.com/zxdawn)
 - [Yufei Zhu (yufeizhu600)](https://github.com/yufeizhu600)
+- [Manuel Carranza (manucarran)](https://github.com/manucarran)
```

Comment:
Please put you name in alphabetical order in the list.

Reply:
Thanks for the tip. Done.